core/state_transition.go

```diff
@@ -389,22 +389,28 @@ func (st *StateTransition) applyCreateValidatorTx(createValidator *staking.Creat
 		return err
 	}
 
-	delegations := []staking.Delegation{
+	zero := big.NewInt(0)
+	wrapper := staking.ValidatorWrapper{}
+	wrapper.Validator = *v
+	wrapper.Delegations = []staking.Delegation{
 		staking.NewDelegation(v.Address, createValidator.Amount),
 	}
-	wrapper := staking.ValidatorWrapper{*v, delegations}
+	wrapper.Snapshot.Epoch = zero
```

Comment:
if you like to add epoch, better add here. But I don't see any use of it yet.

Reply:
hmm, i thought i remember hearing you say that we might want to hold onto it for state sync? and i was using it for the temp sanity check that later would remove once know things stable, correct.